testing/tests/logging.zeek

```diff
@@ -0,0 +1,141 @@
+# @TEST-DOC: Startup a NATS cluster by hand, testing basic logging and node_up and node_down events.
+#
+# @TEST-SERIALIZE: nats
+#
+# @TEST-EXEC: chmod +x ./check-cluster-log.sh
+#
```

Comment:
Like discussed offline, could you add `nats-server` to the "CI setup", and start it up here before the tests here and in `two-nodes.zeek`?